commands/manifest/template.go

```diff
@@ -118,3 +109,50 @@ func templateCmd(c *cli.Context) error {
 
 	return nil
 }
+
+func GetTemplatesManifests(m dx.Manifest) (string, error) {
+
+	switch {
+	case m.Manifests != "" && m.Chart.Name != "":
```

Comment:
Another end user thing is that if we check for `m.Chart.Name != ""` then the user will always have to provide the `chart.name` in their gimlet manifest.

Ideally we should make https://github.com/gimlet-io/gimletd/blob/main/dx/manifest.go#L20 a `*Chart` type, meaning it is a pointer that can have a `nil` value in go. Resulting that the `chart` field will be optional in yaml.

Reply:
We could merge it like this, then a follow up PR will make this change.

Reply:
Sounds good, we can do it in a follow up